### PR TITLE
Update releases-updating.md

### PR DESCRIPTION
**Pull request by JMNordeste-dev**

### CODE REVIEW

File: docs/vendor/releases-updating.md [hunk 1]
```diff
@@ -28,6 +28,19 @@ To update a release:
...
 
+## Archiving Releases
+
+You can archive releases and remove them from view on the Releases page. Any archived releases will still be visible on the Channel's Release History page. 
+
+**Note:** Archiving a release that has been promoted will note remove the release from the Channel's Release History page or prevent app manager from downloading the archived release. 
+
+To archive one or more releases: 
+
+1. Click the 'Archive Multiple Release Icon' on the 'Releases' page.
```
Comment (JustinN-Replicated) [inline docs/vendor/releases-updating.md:39]: I don't know if this is what we want to call this icon (it looks like a list superimposed on a trash can). I had taken this name from the name of the object in the code.

**Comment by jonquil2002** [comment]
@markpundsack  I'm not sure of this doc story that Justin started bc I don't see any archive release option in the vendor portal. Is that feature done? Maybe I'm looking in the wrong place. I'd like to do a final review/edit on this PR and merge it if it's ready from a technical perspective.

**Comment by markpundsack** [comment]
@jonquil2002 It's the little trashcan in the top right. I see it on Replicated QA org. I assume it's available for every team.

image

**Review by jonquil2002 — APPROVED** [review]
lgtm with my changes

**Review by jonquil2002 — APPROVED** [review]
lgtm with my changes.

**Pull request merged**